src/utils/array.js

```diff
@@ -0,0 +1,3 @@
+export const getIntersection = (array1, array2) => {
+  return array1.filter((value) => array2.includes(value));
```

Comment:
하리의 코멘트를 읽고 살펴보니 array1를 순회하면서 array2도 중첩해서 순회하는 형태였네요..!

```suggestion
export const getIntersection = (array1, array2) => {
  const arraySet = new Set(array2);
  return array1.filter((value) => arraySet.has(value));
```
이렇게 array2을 `Set`으로 생성하고 array1의 타겟 요소가 array2 set에 존재하는 값인지 `has` 메서드로 체크한다면 한 번의 순회로 개선할 수 있을 것 같습니다. 힌트 감사합니다 ㅎㅎ